maven-release-manager/src/main/java/org/apache/maven/shared/release/phase/AbstractMapVersionsPhase.java

```diff
@@ -305,11 +300,11 @@ else if ( releaseDescriptor.isBranchCreation() )
 
                 if ( releaseDescriptor.isInteractive() )
                 {
-                    if ( messageKey == null )
+                    if ( messageFormat == null )
                     {
-                        messageKey = getMapversionPromptKey( releaseDescriptor );
+                        messageFormat = getMapversionPromptFormat( releaseDescriptor );
                     }
-                    String message = MessageFormat.format( resourceBundle.getString( messageKey ), project.getName(),
+                    String message = MessageFormat.format( messageFormat, project.getName(),
```

Comment:
I think this can be a `String#format()`.

Reply:
ok, will simplify